.eslintrc.js

```diff
@@ -1,3 +1,8 @@
 const neutrino = require('./packages/neutrino');
 
-module.exports = neutrino(require('./.neutrinorc')).eslintrc();
+module.exports = neutrino({
+  use: require('./.neutrinorc'), // eslint-disable-line global-require
```

Comment:
Perhaps we should move the `require()` to the top of the file and save needing the `eslint-disable-line`? (in this file and the others)

Reply:
Sure thing.